pom.xml

```diff
@@ -55,10 +55,10 @@
 		<legacyuiVersion>1.6.0</legacyuiVersion>
 		<locationbasedaccessVersion>0.1.1</locationbasedaccessVersion>
 		<metadatadeployVersion>1.11.0</metadatadeployVersion>
-		<ugandaemrVersion>3.0.3</ugandaemrVersion>
+		<ugandaemrVersion>3.0.4-SNAPSHOT</ugandaemrVersion>
 		<uiframeworkVersion>3.16.0</uiframeworkVersion>
 		<patientqueueingVersion>1.0.0</patientqueueingVersion>
-        <ugandaemrSyncVersion>1.0.3</ugandaemrSyncVersion>
```

Comment:
@solemabrothers no need to update sync at this time until there is need to

Reply:
@ssmusoke this is resolved